### ISSUE TITLE
How to set JSHint configs?

### ISSUE THREAD

**Issue by MtDalPizzol** [AtomLinter/linter-jshint#90]
I'm using linter-jshint, which works great. The problem is that my project is a Node one and the linter throws warnings on every "require", "module" and, well, every global variable or statement. JSHint has a ```{ "node": true }``` option which should solve the problem, preventing the linter from throwing this warnings. I have tried a bunch of alternatives to set JSHint options and none of them have worked out.

My last guess was to add a ```jshintConfig: {}``` to my package.json file.

Is there, in fact, a option to configure JSHhint through the linter-jshint package?

**Comment by Eschon** [comment]
I think you can configure linter-jshint by putting the options into a file named `.jshintrc` in your project folder.

**Comment by MtDalPizzol** [comment]
A already tried this too. =/
